### PR TITLE
fix(changelog): Update CHANGELOG for release version 0.1.0

### PR DESCRIPTION
**Pull request by dvandra**
Update CHANGELOG for release version 0.1.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
 # Change Log
 
+## 0.0.14 (Jully 2nd, 2020)
+
+- enhancement - Using vulnerability data from Snyk.
```
Comment (arajkumar) [inline CHANGELOG.md:5]: Please add hyper link to Synk.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
 # Change Log
 
+## 0.0.14 (Jully 2nd, 2020)
```
Comment (prashbnair) [inline CHANGELOG.md:3]: Can we change this version 0.1.0

Comment (mathur07) [reply CHANGELOG.md:3]: Spell check Jully -> July

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,35 @@
...
+
+- enhancement - Using vulnerability data from Snyk.
+- enhancement - Includes latest updated security vulnerabilities of `maven`, `node` and `pypi` ecosystems, also includes Snyk curated Security Advisories/Vulnerability Unique to Snyk.
```
Comment (yzainee) [inline CHANGELOG.md:6]: enhancement - Includes latest updated security vulnerabilities for `maven`, `node` and `pypi` ecosystems. It includes Snyk curated Security Advisories/Vulnerabilities, which are not available publicly.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,35 @@
...
+- enhancement - Updated fabric8-analytics-lsp-server to latest version (v[0.2.1](https://www.npmjs.com/package/fabric8-analytics-lsp-server/v/0.2.1)): See [#381](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/381)
+    - Different Underline colors for commonly known vulnerabilities and vulnerability unique to snyk. See [#118](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/118/files#diff-afd7474282d01269197c1d3f05651761R166-R172)
```
Comment (yzainee) [inline CHANGELOG.md:8]: Underline colors -> underlined colour scheme

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,35 @@
...
+    - Updated Diagnostic Message: See [#118](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/118)
+        - Number of Known Security Vulnerabilities and Security Advisories for each dependency.
+        - Highest Severity of vulnerabilities for each affected Dependency. (`Low`/`Medium`/`High`/`Critical`)
+        - Recommended version for Dependencies having Known Security Vulnerabilities.
```
Comment (yzainee) [inline CHANGELOG.md:12]: Dependencies -> dependencies

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,35 @@
...
+        - Added Snyk attribution “Powered by Snyk” in the source of the Diagnostic. See [#121](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/121)
```
Comment (yzainee) [inline CHANGELOG.md:13]: Diagnostic -> diagnostic message.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,35 @@
...
+        - Removed CVE-IDs from message.
+- enhancement - Upadated Stack Report UI: See [#142](https://github.com/fabric8-analytics/fabric8-analytics-stack-report-ui/pull/142)
+    - Updated Security Issue Card content: See [#142](https://github.com/fabric8-analytics/fabric8-analytics-stack-report-ui/pull/142/commits/37d61cf99c1e198c7f85d004b5009ef35d99ff9c)
+        - New headers for Security Issue.
+        - Added Transitives along with the Direct Dependency.
```
Comment (yzainee) [inline CHANGELOG.md:18]: Transitive deps were being marked for vulnerability earlier as well

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,35 @@
...
+        - Separate tabs for Commonly Known Vulnerabilities and Vulnerabilities Unique to Snyk.
+        - Added Snyk Vulnerability ID in place of CVE-ID.
+        - Added Vulnerability Titles with Severity (`Low`/`Medium`/`High`/`Critical`) and removed Tags.
+        - Added Hyperlink to package name, Snyk Vulnerability ID, and Vulnerability Titles.
```
Comment (yzainee) [inline CHANGELOG.md:22]: Hyperlink -> hyperlink

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,35 @@
...
+    - Dependency Details card rearranged in order of preference.
+    - Attribution to Snyk “Powered by Snyk” with a `Sign UP`/`Sign In` Hyperlink to Snyk.
```
Comment (yzainee) [inline CHANGELOG.md:24]: Hyperlink to Snyk->hyperlink to Snyk login page.

File: CHANGELOG.md (hunk 7 above)
Comment (dvandra) [reply CHANGELOG.md:18]: I mean to say we added Transitive as a sub-tab of direct dependency, I will try to justify it properly.

File: CHANGELOG.md (hunk 6 above)
Comment (dvandra) [reply CHANGELOG.md:13]: Diagnostic has a separate filed for source, which is the by default part of each diagnostic.

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,27 +1,26 @@
...
         - Highest Severity of vulnerabilities for each affected Dependency. (`Low`/`Medium`/`High`/`Critical`)
-        - Recommended version for Dependencies having Known Security Vulnerabilities.
+        - Recommended version for dependencies having Known Security Vulnerabilities.
         - Added Snyk attribution “Powered by Snyk” in the source of the Diagnostic. See [#121](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/121)
         - Removed CVE-IDs from message.
 - enhancement - Upadated Stack Report UI: See [#142](https://github.com/fabric8-analytics/fabric8-analytics-stack-report-ui/pull/142)
     - Updated Security Issue Card content: See [#142](https://github.com/fabric8-analytics/fabric8-analytics-stack-report-ui/pull/142/commits/37d61cf99c1e198c7f85d004b5009ef35d99ff9c)
         - New headers for Security Issue.
-        - Added Transitives along with the Direct Dependency.
+        - Added Transitives as a sub-tab in the perticular Direct Dependency.
```
Comment (prashbnair) [inline CHANGELOG.md:17]: "particular"

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,27 +1,26 @@
 # Change Log
 
-## 0.0.14 (Jully 2nd, 2020)
+## 0.1.0 (July 2nd, 2020)
 
-- enhancement - Using vulnerability data from Snyk.
-- enhancement - Includes latest updated security vulnerabilities of `maven`, `node` and `pypi` ecosystems, also includes Snyk curated Security Advisories/Vulnerability Unique to Snyk.
+- enhancement - Integration with [Snyk Intel Vulnerability DB](https://snyk.io/product/vulnerability-database/). It adds value with the latest, fastest and more number of vulnerability coverage derived from numerous sources. It also includes Snyk curated unique and pre-published security advisories that come with early detection and faster identification of vulnerabilities. Snyk Intel Vulnerability DB is the most advanced and accurate open source vulnerability database in the industry. Continuously curated by an experienced Security Research Team, the Snyk Intel Vulnerability DB maintains its high standards which enable your teams to be optimally efficient at containing open source security issues while maintaining their focus on development.
```
Comment (prashbnair) [inline CHANGELOG.md:5]: I am not very sure if we need all that text from Snyk Intel Vulnerability DB

Comment (prashbnair) [inline CHANGELOG.md:5]: It adds value with the latest, fastest and more number of vulnerability coverage derived from numerous sources

It adds value with the latest, fastest and more number of vulnerabilities derived from numerous sources

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,27 +1,26 @@
 # Change Log
 
-## 0.0.14 (Jully 2nd, 2020)
+## 0.1.0 (July 2nd, 2020)
```
Comment (prashbnair) [inline CHANGELOG.md:3]: Should the data be July 14th?

Comment (dvandra) [reply CHANGELOG.md]: @prashbnair PTAL

**Pull request merged**